controlnext/admin.py

```diff
@@ -54,14 +55,8 @@ class BasinAdmin(GeoModelAdmin):
     )
 
 class UserProfileAdmin(admin.ModelAdmin):
-    fieldsets = (
-        (None, {
-            'classes': ['wide'],
-            'fields': (
-                'grower', 'user'
-            )
-        }),
-    )
+    list_display = ('user', )
+    filter_horizontal = ('grower', )
```

Comment:
Jij hoeft niet meer naar m'n admin presentatie van vanmiddag te komen :+1: 

Voor de betere leesbaarheid ben ik zelf a/h overgaan op vierkante haakjes (dus lijsten i.p.v. tuples). Django doet dat zelf ook in de settings, bijvoorbeeld.